pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-ssot"
-version = "1.1.2"
+version = "1.2.0"
```

Comment:
Please don't pre-emptively bump the version in a PR to develop. We should only bump the version when we're preparing to do a release.

Reply:
Was helping to get something out the door. I prefer it come from the PR that is putting things out personally. I do recognize this and will revert.